src/fluree/db/time_travel.cljc

```diff
@@ -155,18 +155,13 @@
           flakes (some-> db
                          dbproto/-rootdb
                          (query-range/index-range :post
-                                                  > [const/$_commit:time 0]
-                                                  < [const/$_commit:time epoch-datetime])
+                                                  >= [const/$_commit:time epoch-datetime]
```

Comment:
I actually think this logic would require just `>`, as a perfectly equal timestamp should return the 't' value of that match.